backend/Dockerfile

```diff
@@ -44,6 +44,7 @@ WORKDIR /bin
 
 COPY --from=builder /opt/app-root/src/backend/src/apiserver/config/ /config
 COPY --from=builder /bin/apiserver /bin/apiserver
+COPY --from=builder /opt/app-root/src/backend/src/apiserver/ilab_pipeline/pipeline.yaml /pipelines/
```

Comment:
If we're going with this method of including the pipeline we should also include the iris pipeline similarly.  This may be out of scope for the ticket, but let's open a story to do so before we forget

Reply:
following https://github.com/opendatahub-io/data-science-pipelines/pull/120/files#r1884437176, we could just COPY the entire directory rather than pinning individual files, so something like:

`COPY --from=builder /opt/app-root/src/backend/src/apiserver/sample_pipelines /pipelines/`